flask_resty/view.py

```diff
@@ -107,8 +108,62 @@ def validate_request_id(self, data, expected_id):
         if id != expected_id:
             raise ApiError(409, {'code': 'invalid_id.mismatch'})
 
+    def update_spec(self, path, spec, many=False, tag=False, **kwargs):
```

Comment:
Maybe `ApiSpecMixin` and `ModelApiSpecMixin`? It's just that this feels in some sense like "extra" behavior that should be opt-in.